meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -468,6 +468,7 @@ MACHINE_FEATURES:remove:stm32mp1common = "fw-update"
 PREFERRED_PROVIDER_u-boot-default-script:stm32mp1common = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script:sota:stm32mp1common = "u-boot-ostree-scr-fit"
 ## stm32mp15-eval
+BOOTDEVICE_LABELS:stm32mp15-eval += "emmc"
```

Comment:
Using append is better imo 
```
BOOTDEVICE_LABELS:append:stm32mp15-eval = " emmc"
```

Reply:
Yeah, since it gets set as 'BOOTDEVICE_LABELS' on the machine config.

Reply:
fixed